### DISCUSSION TITLE
Map view

### DISCUSSION THREAD

**Discussion by eemila** [po8klasie/po8klasie#82] [UX-QA]
Browser QA: 
1. Breadcrumbles are missing
2. How about changing the position of the map and making it 50%vw? 
Sth like this: 
(![image](https://user-images.githubusercontent.com/40060413/98998478-7a955200-2536-11eb-98e7-14c22b067566.png))
It is hard to handle right now

**Comment by micorix** [comment]
@eemila you may want to check ([this](https://github.com/WarsawLO/warsawlo-2.0/issues/21)) out :) (solution prepared by @karoli-na which seems to be similar to this one.)

**Comment by eemila** [comment]
Thanks :)
I think @karoli-na's idea is better but more complex and it can be difficult to implement for 1st release. My idea is just repositioning and it supposed to be a temporary solution before we manage to develop this view :) I mean, my point is - we can not leave the map like this ;)

**Comment by micorix** [comment]
Yep, it may be quite difficult to implement for the initial release... But on the other hand it is the core feature of the app. 

Considering the switch from temporary solution to the hybrid one (@karoli-na's) we would need to figure out whether give the user the ability to toggle views (grid, map, hybrid map) and if not on what stage should we remove it (temp solution, hybrid one).

I really like both solutions but another my concern is that they are keeping map relatively small. We can add "make this map full screen" button but would it be enough? :thinking:

**Comment by karoli-na** [comment]
Or we can make this headline much smaller (or even get rid of it), push the filters and the legend up (+ button to toggle map/grid view). Sth like here, but without these thumbnails on the right: 
![image](https://user-images.githubusercontent.com/66380837/99194343-10bbb900-277f-11eb-8ff2-aa0961fa8e39.png)

**Comment by micorix** [comment]
@karoli-na I think it's great idea. Just to be sure, are you describing changes we have to apply to the map view? So we should leave two "toggleable" views (at least for now)?

**Comment by karoli-na** [comment]
@micorix  Yup, these are changes to the map view, but it still will be toggleable. In case we don't go for the hybrid version. I see that this is mostly the case of complexity and time needed for implementation, so I guess decision what to do now is mostly up to you.

**Comment by micorix** [comment]
Ok, so how about leaving it as it is for now? We can gather more info from our users after initial release and then make a decision about implementing hybrid view based on the that feedback.

**Comment by karoli-na** [comment]
Leaving as it is, but applying these changes to make the map bigger?

**Comment by micorix** [comment]
We have sidebar now :)
